easybuild/easyconfigs/n/netCDF-Fortran/netCDF-Fortran-4.4.3-foss-2016a.eb

```diff
@@ -0,0 +1,20 @@
+name = 'netCDF-Fortran'
+version = '4.4.3'
+
+homepage = 'http://www.unidata.ucar.edu/software/netcdf/'
+description = """NetCDF (network Common Data Form) is a set of software libraries 
+ and machine-independent data formats that support the creation, access, and sharing of array-oriented 
+ scientific data."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'pic': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = [
+    'ftp://ftp.unidata.ucar.edu/pub/netcdf/',
+    'ftp://ftp.unidata.ucar.edu/pub/netcdf/old',
```

Comment:
@SimonPinches the existing netCDF-Fortran easyconfig has (only) `https://github.com/Unidata/netcdf-fortran/archive` listed, I think that's the official repository now?